README.md

```diff
@@ -100,6 +100,30 @@ fn does_not_work_on_miri() {
 }
 ```
 
+An exhaustive list of what `miri` does not support is not available, as this could be
+an unbounded set with FFI and more. However `miri` will explicitly tell you when it finds
+something unsupported with an error:
+
+```
+error: unsupported operation: can't call foreign function: mach_timebase_info
+   --> /home/r/.rustup/toolchains/miri/lib/rustlib/src/rust/src/libstd/sys/unix/time.rs:239:13
+    |
+239 |             mach_timebase_info(&mut info);
+    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't call foreign function: mach_timebase_info
+    |
+    = help: this is likely not a bug in the program; it indicates that the program performed an operation that the interpreter does not support
+    = note: inside call to `std::sys::unix::time::inner::info` at /home/r/.rustup/toolchains/miri/lib/rustlib/src/rust/src/libstd/sys/unix/time.rs:156:24
+    = note: inside call to `std::sys::unix::time::inner::Instant::checked_sub_instant` at /home/r/.rustup/toolchains/miri/lib/rustlib/src/rust/src/libstd/time.rs:263:9
+note: inside call to `std::time::Instant::duration_since` at tests/run-pass/time.rs:25:20
+   --> tests/run-pass/time.rs:25:20
+    |
+25  |         let diff = now2.duration_since(now1);
+    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^
+    = note: inside call to `main` at /home/r/.rustup/toolchains/miri/lib/rustlib/src/rust/src/libstd/rt.rs:67:34
+```
+
+If you do not see an error like this, you are able to keep using `miri`!
```

Comment:
TBH I think this spends way too much space on a problem that is mostly self-explaining when it comes up (as the error hopefully says everything you need to know). Sometimes saying too much about something is as confusing/distracting as saying too little about something. In particular "if you don't see an error, everything is all right" should go without saying.

My suggestion would be to simply amend "You can use this to ignore test cases that will fail under Miri because they do things Miri does not support:" with a parenthetical remark like:

> You can use this to ignore test cases that will fail under Miri because they do things Miri does not support (as indicated by "unsupported operation" errors when such an operation is executed):

Do you think that makes sense?

Reply:
Yeah, it's makes sense - I like my readme's and examples to be really explicit and very clear, but I will make the change.

Reply:
Sorry, that soundded rude, it wasn't my intent! I mean that I like my examples really detailed, and "exactly what a user will see" but I can make it more sparse :)